Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN addgroup -g 10001 -S nonroot && adduser -u 10000 -S -G nonroot -h /home/nonr
 # into /sbin/, etc.
 
 # Tini allows us to avoid several Docker edge cases, see https://github.com/krallin/tini.
+# NOTE: If you are using Docker 1.13 (Docker Engine API v1.25) or greater, Tini is included in Docker itself.
+# This includes all versions of Docker CE. To enable Tini, just pass the --init flag to docker run.
```

Comment:
Thanks for the contribution! Perhaps instead of adding this note, you could just add a `NOTE: See https://github.com/hexops/dockerfile#is-tini-still-required-in-2020-i-thought-docker-added-it-natively` instead?

Reply:
okay description is changed to the link now :+1: